bundle/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     alm-examples: |-
       [
         {
-          "apiVersion": "apm.nxiq.com/v1alpha1",
+          "apiVersion": "sonatype.com/v1alpha1",
```

Comment:
`iq` is completely removed from the name?

Reply:
Never mind, read the PR description and things make sense now

Reply:
Since this is all in an "example" text block, I'm not expecting any direct impact. How did you go about testing this change? (not that I'm expecting any errors, just curious what this block is for). If it's not testable or usable, maybe it would be safer & easier to remove it entirely

Reply:
This alm-examples block in the CSV is built from the CR file (which isn't part of the operator bundle), and it is used to build the YAML any time one clicks "Create Instance" for a NexusIQ in the operator. Any time I created an instance in the OpenShift console, I'm using a YAML generated by these values in the aml-examples.